staging/src/k8s.io/controller-manager/options/generic.go

```diff
@@ -75,8 +75,6 @@ func (o *GenericControllerManagerConfigurationOptions) ApplyTo(cfg *cmconfig.Gen
 		return err
 	}
 
-	cfg.Port = o.Port
-	cfg.Address = o.Address
```

Comment:
why are we not plumbing these?

Reply:
After taking a second look, I found out that `cloud-provider` would call this function https://github.com/kubernetes/kubernetes/blob/0e0cc1ead8ae5937a45e8ca9fa68eb300a1a2f50/staging/src/k8s.io/cloud-provider/options/options.go#L152, so I have reverted this change. Thanks for pointing it out.

Reply:
this still looks wrong... did you forget to push?

Reply:
nice catch.. fixed.